src/Compilers/Core/MSBuildTask/Shared/ManagedCompiler.cs

```diff
@@ -73,7 +73,7 @@ public string ChecksumAlgorithm
         }
         
         /// <summary>
-        /// An instrument flag that specifies instrumentation settings.
+        /// An instrument flag that specifies instrumentation kind.
```

Comment:
@drognanar addressed this, clarifying that multiple instrumentation kinds can appear.